src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -277,6 +293,7 @@ private void AddSeriesTokens(Dictionary<string, Func<TokenMatch, string>> tokenH
         {
             tokenHandlers["{Series Title}"] = m => series.Title;
             tokenHandlers["{Series CleanTitle}"] = m => CleanTitle(series.Title);
+            tokenHandlers["{Series Title The}"] = m => TitleThe(series.Title);
```

Comment:
I prefer `TitleThe` over `Title The` because it otherwise creates ambiguity about the chosen separator.